dafny-lang/dafny
# Fix verification prefix name bug

### Changes
- Fix running verifiable symbols when using prefix name modules
- Fix requesting document symbols when using prefix name modules

### Testing
- Add test `VerifyPositionFromFile1AndUriFromFile2WherePrefixModuleIsInBothFiles`
- Add more assertions to test `LoadsSymbolsInPrefixModule`

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>
